sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ## 7.9.0-beta.1 (Unreleased)
 
 ### Features Added
-
+- Replaced creating single thread executor with `Schedulers.boundedElastic()`
```

Comment:
Can you link to the issue like other changelog items?

Reply:
Will do 👍